stock_picking_completion/hooks.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def column_exists(cr, args):
+    column_exists_query = """
+        SELECT column_name
+        FROM information_schema.columns
+        WHERE table_name=%(table)s AND
+        column_name=%(column)s;
+    """
+    cr.execute(column_exists_query, args)
+    return cr.fetchone()
```

Comment:
I put that here: this function already exists in `openupgradelib`: https://github.com/OCA/openupgradelib/blob/94762d1a5a4172b680a7e019fe4d7db44b2bf7af/openupgradelib/openupgrade_tools.py#L36-L44

But I don't know if adding such dependency for this need is a good thing...